appmetrics.properties

```diff
@@ -16,3 +16,7 @@ com.ibm.diagnostics.healthcenter.logging.plugins=info
 
 # Start with method profiling enabled/disabled: on | off
 com.ibm.diagnostics.healthcenter.data.profiling=off
+
+# Allow probes to transmit data to the client: true | false
+# Defaults to false - probe data not sent.
+#com.ibm.diagnostics.healthcenter.probes.transmit=false
```

Comment:
I've made that change. The appmetrics plugin probably shouldn't put it's properties in the healthcenter namespace. We should avoid that to stop properties for different plugins interfering with each other.